local-volume/README.md

```diff
@@ -165,30 +165,68 @@ $ kubectl create -f bootstrapper/deployment/kubernetes/example-storageclass.yaml
 
 ### Step 3: Creating local persistent volumes
 
-#### Option 1: Bootstrapping the external static provisioner
+#### Option 1: Generating Provisioner's daemonset spec and config map by using Helm chart 
 
-This is optional, only for automated creation and cleanup of local volumes. See
-[bootstrapper/](./bootstrapper) and [provisioner/](./provisioner) for details and
-sample configuration files.
+This option uses helm templating to generate the provisioner's DaemonSet and ConfigMap specs.
+The generated specs can be further customized as needed (usually not necessary), and then deployed using kubectl.
 
-1. Create an admin account with cluster admin priviledge:
+**helm template** uses 3 sources of information:
```

Comment:
This section describing the helm layout could go under a separate README under helm

Reply:
done

Reply:
I think this section (all the way down to the 1. install helm) can be moved to helm/readme

Reply:
done